.github/workflows/dependency-image.Dockerfile

```diff
@@ -65,7 +65,12 @@ RUN sdkmanager --install \
     "platform-tools;35.0.2" \
     "platforms;android-35" \
     "ndk;27.0.12077973"
-RUN yes | sdkmanager --licenses
+RUN mkdir -p $ANDROID_HOME/licenses && \
+    touch $ANDROID_HOME/licenses/android-sdk-license && \
+    echo "8933bad161af4178b1185d1a37fbf41ea5269c55" >> $ANDROID_HOME/licenses/android-sdk-license && \
+    echo "d56f5187479451eabf01fb78af6dfcb131a6481e" >> $ANDROID_HOME/licenses/android-sdk-license && \
+    echo "24333f8a63b6825ea9c5514f83c2829b004d1fee" >> $ANDROID_HOME/licenses/android-sdk-license && \
+    yes | sdkmanager --licenses
```

Comment:
Do we still need this?

Reply:
Confirmed build succeeds without these four lines. I'm surprised; I thought AGP would throw an exception without a license agreement. Do you know the reason by any chance?

Reply:
I believe AGP checks that the license has been approved by looking for the various checksums. It's one step more secure than just having a file that says "license-agreement-signed=true".

Reply:
Can you review the changes again? I removed these four lines, and the build has succeeded:
https://github.com/gobley/gobley/actions/runs/14724101738

And the build certainly succeeded without `/android-sdk/licenses/android-sdk-license`, as `cat /android-sdk/licenses/android-sdk-license` fails:
https://github.com/gobley/gobley/actions/runs/14724423698

Reply:
Sorry for the delay! I'll take a look in a few hours.